test/loader.test.js

```diff
@@ -54,10 +57,10 @@ test.cb("should transpile the code snippet", t => {
       t.true(files.length === 1);
       fs.readFile(path.resolve(t.context.directory, files[0]), (err, data) => {
         t.is(err, null);
-        const test = "var App = function App()";
+        const test = "var App = function App(arg)";
```

Comment:
CI was failing because new Babel version generates slightly different outputs. I have updated the test Babel config and ensured that the test Babel config does not inherit root Babel config.